Source/Shared/Library/DefaultCacheConverter.swift

```diff
@@ -37,6 +38,9 @@ public struct DefaultCacheConverter<T> {
 
   /**
    Encodes an instance to NSData
+
```

Comment:
Sometimes I feel the documentation is a bit unnecessary. I can see that by looking at the function signature